src/casp/cell_data_manager/README.md

```diff
@@ -0,0 +1,118 @@
+# Cell Data Manager Module
```

Comment:
Cell Data Manager Module -> Cellarium Data Manager Module

Reply:
also, rename the directory structure, `cell_data_manager` -> `cellarium_data_manager`.